extension/src/plots/webview/contract.ts

```diff
@@ -50,8 +50,13 @@ export type SectionCollapsed = typeof DEFAULT_SECTION_COLLAPSED
 
 export type ComparisonRevisionData = { [revision: string]: ComparisonPlot }
 
+export type ComparisonBoundingBoxLabels = {
+  [label: string]: { selected: boolean; color: string }
+}
+
 export type ComparisonPlots = {
   path: string
+  boundingBoxLabels: ComparisonBoundingBoxLabels
   revisions: ComparisonRevisionData
```

Comment:
I've added two values to the `PlotsComparisonData`. `ComparisonBoundingBoxLabels` inside of `ComparisonPlots` (holds the labels information that exist in that plot/image row) and `ComparisonPlotBoundingBox` (holds box labels/coordinates for each single image).

Reply:
Currently, I was thinking that our backend would manage the "selected" state of the classes, but we could also manage that in redux instead.